corehq/apps/api/tests/core_api.py

```diff
@@ -586,40 +648,29 @@ def test_inserted_at_query(self):
             "/a/test_domain/api/v0.5/form/",
             data={'_search': json.dumps(query)}
         )
-        expected = {
-            "filter": {
-                "and": [
-                    {
-                        "term": {
-                            "domain.exact": "test_domain"
-                        }
-                    },
-                    query['filter']
-                ]
-            }
+        _filter = {
+            "and": [
+                {
+                    "term": {
+                        "domain.exact": "test_domain"
+                    }
+                },
+                {
+                    "term": {
+                        "doc_type": "xforminstance"
+                    }
+                },
+                query['filter'],
+                {
+                    "match_all": {}
+                }
+            ]
         }
         expected = {
             "query": {
-                "filtered": {
-                    "filter": {
-                        "and": [
-                            {
-                                "term": {
-                                    "domain.exact": "test_domain"
-                                }
-                            },
-                            {
-                                "term": {
-                                    "doc_type": "xforminstance"
-                                }
-                            },
-                            query['filter'],
-                            {
-                                "match_all": {}
-                            }
-                        ]
-                    },
-                    "query": {
+                "bool": {
+                    "filter": _filter["and"],
```

Comment:
This appears to be the only place that `_filter` is referenced. It seems like it would be clearer to remove that variable and inline the list here
```suggestion
                    "filter": [
                        {
                            "term": {
                                "domain.exact": "test_domain"
                            }
                        },
                        {
                            "term": {
                                "doc_type": "xforminstance"
                            }
                        },
                        query['filter'],
                        {
                            "match_all": {}
                        }
                    ],
```

Reply:
Will fix it